### PR TITLE
updated UI for Placement Sample App design

### PR DESCRIPTION
**Pull request by ashinagawa**
@sergeyksv @kosyakow 
I updated UI and code to implement our Placement Sample App UI design.
Please review and merge this before the final build.

Thanks!

### CODE REVIEW

File: Sample/Unity Sample Application/Assets/TitleGUI.cs [hunk 1]
```diff
@@ -42,67 +57,56 @@ public class TitleGUI : MonoBehaviour {
...
-			Vungle.loadAd(list[2]);
-		}
-		GUILayout.EndHorizontal();
-
-		GUILayout.EndArea ();
+
+	void SetupButtonsAndText () {
+		placementIdList = new List<string>(placements.Keys);
+
+		appIDText.text = "App ID: " + appID;
```
Comment (kosyakow) [inline Sample/Unity Sample Application/Assets/TitleGUI.cs]: NullReferenceException: Object reference not set to an instance of an object
TitleGUI.SetupButtonsAndText () (at Assets/TitleGUI.cs:64)
TitleGUI.Start () (at Assets/TitleGUI.cs:45)

File: Sample/Unity Sample Application/Assets/TitleGUI.cs [hunk 2]
```diff
@@ -42,67 +57,56 @@ public class TitleGUI : MonoBehaviour {
...
 
-	void initializeGUIStyles() {
-		titleLabelStyle = new GUIStyle ();
-		titleLabelStyle.stretchWidth = true;
-//		titleLabelStyle.stretchHeight = true;
-		titleLabelStyle.fixedWidth = Screen.width;
-//		titleLabelStyle.fixedHeight = headerHeight;
-		titleLabelStyle.alignment = TextAnchor.MiddleCenter;
+	void updateButtonState() {
+		playPlacement1Button.enabled = placements[placementIdList[0]];
```
Comment (kosyakow) [inline Sample/Unity Sample Application/Assets/TitleGUI.cs]: NullReferenceException: Object reference not set to an instance of an object
TitleGUI.SetupButtonsAndText () (at Assets/TitleGUI.cs:64)
TitleGUI.Start () (at Assets/TitleGUI.cs:45)

**Comment by ashinagawa** [comment]
@kosyakow  Because I updated UI in Unity 5, it won't work in Unity 4? 
If I re-do update in Unity 4, it should work in Unity 4 and 5? 
Do you know? Thanks!

**Comment by ashinagawa** [comment]
@sergeyksv @zombierzn Please ignore this brunch.  I will create another branch and work in Unity 4. WIll be done in hour or two.

**Comment by ashinagawa** [comment]
ignore this branch.

**Pull request closed without merging**